docs/api/features/manifold.py

```diff
@@ -69,19 +64,21 @@ def dataset_example(dataset="occupancy", manifold="all", path="images/", **kwarg
         return
 
     # Create single example
-    _, ax = plt.subplots(figsize=(9, 6))
-    oz = Manifold(ax=ax, manifold=manifold, **kwargs)
 
+    _, ax = plt.subplots(figsize=(9, 6))
     if dataset == "occupancy":
         X, y = load_occupancy()
     elif dataset == "concrete":
         X, y = load_concrete()
     else:
         raise Exception("unknown dataset '{}'".format(dataset))
-
-    oz.fit_transform(X, y)
-    oz.show(outpath=path)
-
+    if(quick==True):
+        oz = manifold_embedding(X, y, manifold="isomap", n_neighbors=10)
```

Comment:
We need to pass `ax` and `**kwargs` here! Also, we can specify `manifold=manifold` and pass `isomap` from the function call.